media/js/wiki.js

```diff
@@ -302,6 +302,8 @@
         var $pageButtons = $('.page-buttons');
         var pageButtonsOffset = $pageButtons.offset();
 
+        var stickyFeatureEnabled = $pageButtons.attr('data-sticky') === 'true';
```

Comment:
We could think about removing the `data-sticky` attribute altogether, unless we plan to disable the sticky buttons dynamically in the future.